### PR TITLE
DOC-646: Add documentation for new 'ToggleToolbarDrawer' command and query command state

### PR DESCRIPTION
**Pull request by ltrouton**
Related Ticket: DOC-646

Description of Changes:
* Add documentation for new 'ToggleToolbarDrawer' command and query command state handler

Pre-checks:
- [X] Branch prefixed with `feature/` or `hotfix/`
- [X] ~~`_data/nav.yml` has been updated (if applicable)~~
- [X] ~~Files has been included where required (if applicable)~~
- [X] ~~Files removed have been deleted, not just excluded from the build (if applicable)~~
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: advanced/editor-command-identifiers.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ The commands on the following table are provided by the {{site.productname}} edi
...
 | mceToggleFormat      | Toggles a specified format by name. The value is the name of the format to toggle. For a list of options, see: [Content formatting options - Built-in formats]({{ site.baseurl }}/configure/content-formatting/#built-informats).                                                                               |
 | ToggleSidebar        | Closes the currrent sidebar, or toogles the sidebar if the sidebar name is provided as a value (_`<sidebar-name>`_).                                                                                                                                                                        |
+| ToggleToolbarDrawer  | Toggles the toolbar drawer. For information on toolbars, see: [User interface options - Toolbar]({{site.baseurl}}/configure/editor-appearance/#toolbar).                                                                                                                                                                        |
```
Comment (lnewson) [inline advanced/editor-command-identifiers.md:78]: Does this need the `{{ site.requires_5_5v }}` note, to indicate it only works in 5.5 or above?

File: advanced/editor-command-identifiers.md [hunk 2]
```diff
@@ -460,6 +462,7 @@ The following command states can be queried using the [queryCommandState]({{ sit
 | Strikethrough        | Returns `true` if the content is formatted using the same markup as the {{site.productname}} `Strikethrough` command.          |
 | Subscript            | Returns `true` if the content is formatted using the same markup as the {{site.productname}} `Subscript` command.              |
 | Superscript          | Returns `true` if the content is formatted using the same markup as the {{site.productname}} `Superscript` command.            |
+| ToggleToolbarDrawer  | Returns `true` if the toolbar drawer is open. The state can be controlled by the {{site.productname}} `ToggleToolbarDrawer` command. |
```
Comment (lnewson) [inline advanced/editor-command-identifiers.md:465]: Likewise, does this need the `{{ site.requires_5_5v }}` note?

**Review by lnewson — APPROVED** [review]

File: advanced/editor-command-identifiers.md (hunk 1 above)
Comment (ltrouton) [reply advanced/editor-command-identifiers.md]: Fixed

File: advanced/editor-command-identifiers.md (hunk 2 above)
Comment (ltrouton) [reply advanced/editor-command-identifiers.md]: Fixed

**Review by tylerkelly13 — APPROVED** [review]

**Review by tylerkelly13 — APPROVED** [review]
LGTM. Please check that I didn't ruin it with my release note :)

File: release-notes/release-notes55.md [hunk 3]
```diff
@@ -41,6 +41,15 @@ The new listbox component for custom dialogs is a composite component with a lab
 
 For information on the listbox dialog component, see: [Dialog components - listbox]({{site.baseurl}}/ui-components/dialogcomponents/#listbox).
 
+### The toolbar drawer can now be controlled using commands
```
Comment (ltrouton) [inline release-notes/release-notes55.md:44]: @tylerkelly13  Looks good. Was just wondering whether 'toggle toolbar' should be capitalized or not below? I have it non-capitalized everywhere else and am not if sure if it needs to consistent or if the capitalization is context dependent.

Also, for `The command allow the Toolbar Drawer...` below, I think `allow` should be `allows`?

File: advanced/editor-command-identifiers.md [hunk 4]
```diff
@@ -460,6 +462,7 @@ The following command states can be queried using the [queryCommandState]({{ sit
 | Strikethrough        | Returns `true` if the content is formatted using the same markup as the {{site.productname}} `Strikethrough` command.          |
 | Subscript            | Returns `true` if the content is formatted using the same markup as the {{site.productname}} `Subscript` command.              |
 | Superscript          | Returns `true` if the content is formatted using the same markup as the {{site.productname}} `Superscript` command.            |
+| ToggleToolbarDrawer  | Returns `true` if the toolbar drawer is open. The state can be controlled by the {{site.productname}} `ToggleToolbarDrawer` command. {{site.requires_5_5v}} |
```
Comment (tylerkelly13) [inline advanced/editor-command-identifiers.md:465]:
```suggestion
| ToggleToolbarDrawer  | Returns `true` if the Toolbar Drawer is open. The state can be controlled by the {{site.productname}} `ToggleToolbarDrawer` command. {{site.requires_5_5v}} |
```

File: advanced/editor-command-identifiers.md [hunk 5]
```diff
@@ -75,6 +75,7 @@ The commands on the following table are provided by the {{site.productname}} edi
...
 | mceToggleFormat      | Toggles a specified format by name. The value is the name of the format to toggle. For a list of options, see: [Content formatting options - Built-in formats]({{ site.baseurl }}/configure/content-formatting/#built-informats).                                                                               |
 | ToggleSidebar        | Closes the currrent sidebar, or toogles the sidebar if the sidebar name is provided as a value (_`<sidebar-name>`_).                                                                                                                                                                        |
+| ToggleToolbarDrawer  | Toggles the toolbar drawer. For information on toolbars, see: [User interface options - Toolbar]({{site.baseurl}}/configure/editor-appearance/#toolbar). {{site.requires_5_5v}}                                                                                                             |
```
Comment (tylerkelly13) [inline advanced/editor-command-identifiers.md:78]:
```suggestion
| ToggleToolbarDrawer  | Toggles the Toolbar Drawer. For information on toolbars, see: [User interface options - Toolbar]({{site.baseurl}}/configure/editor-appearance/#toolbar). {{site.requires_5_5v}}                                                                                                             |
```

File: release-notes/release-notes55.md [hunk 6]
```diff
@@ -41,6 +41,15 @@ The new listbox component for custom dialogs is a composite component with a lab
...
+
+A new editor command (`execCommand('ToggleToolbarDrawer')`) and editor state query (`queryCommandState('ToggleToolbarDrawer')`) have been added. The command allow the Toolbar Drawer (which holds excess toolbar buttons) to be opened and closed using editor commands. The current state of the Toolbar Drawer can also be queried.
```
Comment (tylerkelly13) [inline release-notes/release-notes55.md:46]:
```suggestion
A new editor command (`execCommand('ToggleToolbarDrawer')`) and editor state query (`queryCommandState('ToggleToolbarDrawer')`) have been added. The command allows the Toolbar Drawer (which holds excess toolbar buttons) to be opened and closed using editor commands. The current state of the Toolbar Drawer can also be queried.
```

File: release-notes/release-notes55.md (hunk 3 above)
Comment (tylerkelly13) [reply release-notes/release-notes55.md:44]: Did you mean 'toolbar drawer'?

I've added suggestions for you to accept/reject.

**Review by tylerkelly13 — APPROVED** [review]

**Review by EkimChau — APPROVED** [review]

**Pull request merged**